build.html

```diff
@@ -77,7 +77,7 @@
     </script>
   {{/if}}
 {{else}}
-<div class="dynamic-list-placeholder">
+<div class="dynamic-list-placeholder" data-dynamic-lists-id="{{ id }}">
```

Comment:
There's no need to update this.

```suggestion
<div class="dynamic-list-placeholder">
```